package.json

```diff
@@ -60,6 +60,7 @@
     "url": "https://github.com/elastic/kibana.git"
   },
   "dependencies": {
+    "@bigfunger/decompress-zip": "^0.2.0-stripfix2",
```

Comment:
Is that also in master?  If so, can we address that in a follow-up PR?  This is just a backport, and it's already pretty monolithic.